src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -62,6 +62,13 @@ public String toString() {
         return '[' + tagName + ']';
     }
 
+    /**
+     * Returns the String representation of the tag name.
+     */
+    public String getTagName() {
```

Comment:
Is this really needed since tagName is already public?

Reply:
Great catch! 

It would be worrying if tagName was declared as `public` only, as it would mean that other classes can modify `tagName` to another value other than the 3 we have defined (Close Friend, Friend, Emergency). Fortunately, the `final` keyword ensures that it's immutable. 

I will rectify this and it will be reflected in the next commit. Thank you!